lib/google_drive/session.rb

```diff
@@ -102,13 +113,24 @@ def self.from_service_account_key(
     #   scope
     #   scope=
     #   save
+    #
+    # +options+ is a hash which may contain the following keys:
+    #   client_secret: if present, this will overwrite config.client_secret
+    #   client_id: if present, this will overwrite config.client_id
+    #   client_options: a hash or ::Google::Apis::ClientOptions. See
+    #     https://github.com/google/google-api-ruby-client/blob/master/lib/google/apis/options.rb#L37-L58
```

Comment:
Can you link to RDoc instead:
https://www.rubydoc.info/github/google/google-api-ruby-client/Google/Apis/ClientOptions